modules/administration/pages/maintenance-window.adoc

```diff
@@ -1,33 +1,39 @@
 [[maintenance-window]]
-= Maintenance Window Checklist
+= {productname} Maintenance Window
 
-If you work on a {productname} Server with scheduled maintenance windows, you might find it difficult to remember all the things that you need to during that critical downtime.
+If you work with scheduled maintenance windows, you might find it difficult to remember all the things that you need to do before, during, and after that critical downtime of the {productname} Server.
+{productname} Server related systems such as Inter-Server Synchronization Slave Servers or {productname} Proxies are also affected and have do be considered.
+
+{suse} recommends to keep the {productname} infrastructure (servers, proxies, build hosts, etc.) always up-to-date.
```

Comment:
```suggestion
{suse} recommends to keep the {productname} infrastructure (Servers, Proxies, build hosts, etc.) always up-to-date.
```

(I think we capitalize those, right @Loquacity?)

Reply:
yes, to be cap'ed if they appear close with {productname}